common/common/src/main/java/io/helidon/common/LazyValueImpl.java

```diff
@@ -16,42 +16,28 @@
 
 package io.helidon.common;
 
-import java.util.concurrent.locks.Lock;
-import java.util.concurrent.locks.ReentrantLock;
+import java.util.concurrent.atomic.AtomicReference;
```

Comment:
Please revert the state to original implementation + add the new constructor that sets the `value` and sets `loaded` to true.

Reply:
reverted